### PR TITLE
Added a dedicated settings page and custom background feature

### LINKED ISSUE #146: To allow the user to have a custom background
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/susi_server

**Current behavior:**
Only 2 restricted themes available ,the ability to the user to atleast be able to add a custom background

**Would you like to work on the issue?**
Yes
```

### PR DESCRIPTION
**Pull request by stealthanthrax**
Fixes #146 via #187 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
A new dedicated settings page has been added and custom background feature is also added.

#### Changes proposed in this pull request:
screen shot 2017-11-30 at 5 15 43 pm
screen shot 2017-12-01 at 12 25 49 am

### CODE REVIEW

File: src/script.js [hunk 1]
```diff
@@ -18,6 +18,16 @@ var upCount = 0;
...
 var storageItems= [];
 var storageArr = [];
+var save = document.getElementById("Save");
+var backUrl = localStorage.getItem("theValue");
+var box = document.getElementById("box");
+save.addEventListener("click",()=>{
+    box.style.backgroundImage = "url("+backUrl+")";
+    box.style.backgroundRepeat = "no-repeat";
+    box.style.backgroundSize = "cover";
+//    alert("Settings Applied Successfully");
```
Comment (gabru-md) [inline src/script.js]: Comments must be added only in case of documentation.
Since there is none. Please remove the commented line.
Thanks!

**Comment by stealthanthrax** [comment]
@gabru-md ,done

**Comment by stealthanthrax** [comment]
@zamhaq , to apply the changes by the user in the settings page

**Comment by zamhaq** [comment]
@stealthanthrax why there? Why can't it be done from the settings page?

**Comment by gabru-md** [comment]
Please look into it.
@stealthanthrax

**Comment by stealthanthrax** [comment]
@zamhaq , @ms10398 ,i can try and use something like "onload " with the body tag

**Comment by stealthanthrax** [comment]
@zamhaq @gabru-md @ms10398 ,please have a look .

**Comment by 0x48piraj** [comment]
@zamhaq I think we should merge https://github.com/fossasia/susi_chromebot/pull/205
And, then apply this change, right ?

**Pull request merged**